### PR TITLE
feat: add abandoned_time_limit to PublisherStalledCheck

### PR DESCRIPTION
**Pull request by cctdaniel**

### CODE REVIEW

File: pyth_observer/check/publisher.py [hunk 1]
```diff
@@ -221,7 +221,7 @@ def error_message(self) -> dict:
             "symbol": self.__state.symbol,
             "publisher_price": f"{self.__state.price} ± {self.__state.confidence_interval}",
             "aggregate_price": f"{self.__state.price_aggregate} ± {self.__state.confidence_interval_aggregate}",
-            "deviation": deviation,
+            "deviation": f"{deviation:.2f}%",
```
Comment (cctdaniel) [inline pyth_observer/check/publisher.py:224]: drive-by: format string here instead of `event.py`

**Review by ayazabbas — APPROVED** [review]

File: sample.config.yaml [hunk 2]
```diff
@@ -46,7 +46,8 @@ checks:
       max_aggregate_distance: 6
     PublisherStalledCheck:
       enable: true
```
Comment (ali-bahjati) [inline sample.config.yaml:48]: you could set the `enable` to false here actually to disable the check.

**Review by ali-bahjati — APPROVED** [review]

File: sample.config.yaml (hunk 2 above)
Comment (cctdaniel) [reply sample.config.yaml:48]: well the reason for this is to test it for a couple of symbols first (since the less liquid ones are noisy), if we set it to false then all will be disabled

Comment (ali-bahjati) [reply sample.config.yaml:48]: Hmmm i think this is because if this be `false` then other checks should override it and say `enable: yes`. But that's clear now.
